components/tag/index.css

```diff
@@ -12,195 +12,190 @@
-	--spectrum-tag-content-color-emphasized: var(--spectrum-white);
-
-	/* disabled variant content */
-	--spectrum-tag-content-color-disabled: var(--spectrum-disabled-content-color);
-}
+	--spectrum-tag-background-color-emphasized: var(--highcontrast-tag-background-color-emphasized, var(--mod-tag-background-color-emphasized, var(--spectrum-accent-background-color-default)));
+	--spectrum-tag-background-color-emphasized-hover: var(--highcontrast-tag-background-color-emphasized-hover, var(--mod-tag-background-color-emphasized-hover, var(--spectrum-accent-background-color-hover)));
+	--spectrum-tag-background-color-emphasized-active: var(--highcontrast-tag-background-color-emphasized-active, var(--mod-tag-background-color-emphasized-active, var(--spectrum-accent-background-color-down)));
+	--spectrum-tag-background-color-emphasized-focus: var(--highcontrast-tag-background-color-emphasized-focus, var(--mod-tag-background-color-emphasized-focus, var(--spectrum-accent-background-color-key-focus)));
+	--spectrum-tag-content-color-emphasized: var(--highcontrast-tag-content-color-emphasized, var(--mod-tag-content-color-emphasized, var(--spectrum-white)));
+
+	/* disabled variant */
+	--spectrum-tag-background-color-disabled: var(--highcontrast-tag-background-color-disabled, var(--mod-tag-background-color-disabled, var(--spectrum-disabled-background-color)));
+	--spectrum-tag-content-color-disabled: var(--highcontrast-tag-content-color-disabled, var(--mod-tag-content-color-disabled, var(--spectrum-disabled-content-color)));
 
-.spectrum-Tag,
-.spectrum-Tag--sizeM {
+	/* default size/sizeM */
 	--spectrum-tag-height: var(--spectrum-component-height-100);
+	--spectrum-tag-min-inline-size: var(--spectrum-tag-minimum-width-medium);
 	--spectrum-tag-font-size: var(--spectrum-font-size-100);
 	--spectrum-tag-icon-size: var(--spectrum-workflow-icon-size-100);
+	--spectrum-tag-corner-radius: var(--spectrum-corner-radius-medium-size-medium);
 
-	--spectrum-tag-clear-button-spacing-inline-start: var(--spectrum-text-to-visual-100);
+	--spectrum-tag-clear-button-size: var(--spectrum-cross-icon-size-100);
 	--spectrum-tag-clear-button-spacing-block: var(--spectrum-tag-top-to-cross-icon-medium);
+	--spectrum-tag-label-to-clear-icon: var(--spectrum-tag-label-to-clear-icon-medium);
+	--spectrum-tag-edge-to-clear-icon: var(--spectrum-tag-edge-to-clear-icon-medium);
+
+	--spectrum-tag-visual-spacing-inline-start: var(--spectrum-component-edge-to-visual-100);
+	--spectrum-tag-visual-spacing-inline-end: var(--spectrum-text-to-visual-100);
 
 	--spectrum-tag-icon-spacing-block-start: var(--spectrum-component-top-to-workflow-icon-100);
 	--spectrum-tag-icon-spacing-block-end: var(--spectrum-component-top-to-workflow-icon-100);
-	--spectrum-tag-icon-spacing-inline-end: var(--spectrum-text-to-visual-100);
 
-	--spectrum-tag-avatar-spacing-block-start: var(--spectrum-tag-top-to-avatar-medium); /* 6px 9px */
+	--spectrum-tag-avatar-size: var(--spectrum-avatar-size-100);
```

Comment:
Avatar and thumbnail size don't correspond to the usual 75 = small, 100 = medium, 200 = large sizing conventions.

Reply:
It does fit better with medium and it looked like this avatar size is how React has it, so I changed it and moved it to my list of things to ask about, but to be fair, the avatar's margin is still slightly (2px I think) out of bounds in the large tag: 
![image](https://github.com/user-attachments/assets/a7198c6f-8e3f-46c1-a684-dbe2737ec6b1)